### ISSUE TITLE
Generated XPath's do not start with //

### ISSUE THREAD

**Issue by ORESoftware** [johannhof/xpath-dom#4]
Looks like the unique xpath's generated from this call:

https://github.com/johannhof/xpath-dom#xpathgetuniquexpathnode--relativeto

generates an xpath like this `'span[1]'` =>

```js
assert.equal('span[1]', xpath.getUniqueXPath(xpath.find("div/h1/span"), xpath.find("div/h1")));
```
but I would have hoped it would generate a relative xpath that starts with two slashes, so it would look something like like:

`'//span[1]'`

**Comment by johannhof** [comment]
So, it's been a while since I wrote this, but I think I remember leaving it out on purpose. In any case, I'd happily take a patch for this, I think. This doesn't strike me as breaking backwards compatibility.

**Comment by ORESoftware** [comment]
yes I think a relative xpath always starts with `//`, but I'd have to double check
